### DISCUSSION TITLE
Corner Radius Fills?

### DISCUSSION THREAD

**Discussion by SafariKC** [tesla-android/issue-tracker#122] [Ideas]
This project is pretty great.  I'm curious if you have at all considered doing a pixel sample every once and a while on the corner radius and trying to fill the black area with something that hides the rounded corners seeing that the Tesla UX is very square?

**Comment by mikegapinski** [comment]
I was considering this for a long time, just haven't had the time to do it. It's possible with the new video layer, I'm switching video rendering to flutter soon, this'll make frame sampling possible and easy to implement

**Reply by SafariKC** [reply]
Excellent! Thanks for all you do my friend!
